src/client/eppo-client.ts

```diff
@@ -309,6 +321,32 @@ export default class EppoClient {
     );
   }
 
+  /**
+   * Initializes the `EppoClient` from the provided configuration. This method is async only to
+   * accommodate writing to a persistent store. For fastest initialization, (at the cost of persisting configuration),
+   * use `bootstrap` in conjunction with `MemoryOnlyConfigurationStore` instances which won't do an async write.
+   */
+  bootstrap(configuration: IConfigurationWire): void {
+    if (!configuration.config) {
+      throw new Error('Flag configuration not provided');
+    }
+    const flagConfigResponse: IUniversalFlagConfigResponse = inflateResponse(
+      configuration.config.response,
+    );
+    const banditParamResponse: IBanditParametersResponse | undefined = configuration.bandits
+      ? inflateResponse(configuration.bandits.response)
+      : undefined;
+
+    // This method runs async because the configuration stores potentially have an async write at the end of updating
+    // the configuration. Most instances of offlineInit will use `MemoryOnlyConfigurationStore` instances which actually
+    // accomplish the config write synchronously.
+    // `void` keyword here suppresses warnings about leaving the promise hanging.
```

Comment:
alternatively you could do a `.catch((err) => logger.error('Error hydrating configuration store', err)` I reckon that may be preferable to having a potentially unhandled rejection which could tank parent process